### PR TITLE
fix contribution guidelines link on readme

### PR DESCRIPTION
**Pull request by chrisns**
👋 hey, just figured I'd update this for you.
Also looking at a bunch of the PRs you've closed say this is a 'generated repo' so changes (inc maybe this one) don't get accepted.
It'd be great to update the contributing section with some steer of how to actually influence this repo, or if its not worthwhile at all, some projects state there should be an issue prior to a PR, which might save folk wasting time. anyway, just a thought. thanks!

### CODE REVIEW

**Comment by frenck** [comment]
This part should be removed actually, as it is a generated repository nowadays.

**Comment by chrisns** [comment]
@frenck righto, I've updated the PR in line with that, inc updating the PR template to signpost it a bit better

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,29 +0,0 @@
-# Contributing
```
Comment (frenck) [inline .github/CONTRIBUTING.md:1]: This file is also used for the add-on page. Removing this breaks that.

**Comment by chrisns** [comment]
Closing this pr

Seems like you've got some other things to fix around making exemptions to rules you have on repos that aren't autogenerated.
Can't see obviously see the code you're using for any of that, so can't help PR that first.

All the best.

**Comment by frenck** [comment]
Ok bye 👋  Thanks for trying though, that is appreciated 👍

**Comment by chrisns** [comment]
I started by updating a link on the readme.
you clarified that people can't influence this repo at all, so I made updates to reflect that more clearly
I'm probably being thick and slow and can't see what is actually generating the repo
I'm a happy end consumer of this repo so no hard feelings

**Comment by frenck** [comment]
>  so no hard feelings

Not from my end either, I will take a look when the December release is out the door.
I agree this is incorrect.

**Pull request closed without merging**